src/encoded/static/components/body_map.js

```diff
@@ -856,53 +885,58 @@ BodyMap.contextTypes = {
 // Button to display the actual body map facet <BodyMapModal>
 export const ClickableThumbnail = (props) => {
     // "toggleThumbnail" toggles whether or not the pop-up is displayed
-    const { toggleThumbnail, organism } = props;
+    const { toggleThumbnail, organism, context } = props;
     let CellsList = {};
     if (props.organism === 'Homo sapiens') {
         CellsList = HumanCellsList;
     } else if (props.organism === 'Mus musculus') {
         CellsList = MouseCellsList;
     }
+
     return (
-        <button
-            type="button"
-            className="body-image-thumbnail"
-            onClick={() => toggleThumbnail()}
-        >
-            <div className="body-map-expander">Filter results by body diagram</div>
-            {svgIcon('expandArrows')}
-            {organism === 'Homo sapiens' ?
-                <HumanBodyDiagram
-                    BodyList={HumanList}
-                />
-            : null}
-            {organism === 'Mus musculus' ?
-                <MouseBodyDiagram
-                    BodyList={MouseList}
-                />
-            : null}
-            <div className="body-list body-list-narrow">
-                <ul className="body-list-inner">
-                    {Object.keys(CellsList).map((image) => (
-                        <div
-                            className={`body-inset ${image}`}
-                            id={image}
-                            key={image}
-                        >
-                            <img className="active-image" src={`/static/img/bodyMap/insetSVGs/${organism === 'Mus musculus' ? 'mouse_' : ''}${image.replace(' ', '_')}.svg`} alt={image} />
-                            <img className="inactive-image" src={`/static/img/bodyMap/insetSVGs/${organism === 'Mus musculus' ? 'mouse_' : ''}${image.replace(' ', '_')}_deselected.svg`} alt={image} />
-                            <div className="overlay" />
-                        </div>
-                    ))}
-                </ul>
-            </div>
-        </button>
+        <>
+            <button
+                type="button"
+                className="body-image-thumbnail"
+                onClick={() => toggleThumbnail()}
+            >
+                <div className="body-map-expander">Filter results by body diagram</div>
```

Comment:
This may be an old issue or not any issue at all. On laptops, I cannot view the bottom of the body-modal

Reply:
I added some margin at the bottom of the modal because I always meant to have some margin there - it looks a little suboptimal to have it lined up with the bottom of the page. That's not really the issue you were bringing up here but it's adjacent to this issue!